ui/admin/tests/e2e/README.md

```diff
@@ -1,36 +1,133 @@
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Admin UI e2e tests
+
+- [Admin UI e2e tests](#admin-ui-e2e-tests)
+  - [Prerequisites:](#prerequisites)
+    - [Accesses](#accesses)
+    - [Software](#software)
+  - [Getting Started](#getting-started)
+    - [Setup Boundary CLI](#setup-boundary-cli)
+    - [Setup Boundary UI](#setup-boundary-ui)
+    - [Setup EC2 instance:](#setup-ec2-instance)
+    - [Setup HCP Terraform (Terraform cloud):](#setup-hcp-terraform-terraform-cloud)
+    - [Setup Enos:](#setup-enos)
+  - [Run tests:](#run-tests)
+    - [Run Enos Scenario](#run-enos-scenario)
+    - [Run tests](#run-tests)
+    - [Destroy Enos Scenario](#destroy-enos-scenario)
+  - [Developing Tests](#developing-tests)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 # Admin UI e2e tests
 
 This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
 variety of environments as long as the appropriate environment variables are set. You can choose to
 spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
-to generate it for you.
+to generate it for you. More information about [Enos available here.](https://github.com/hashicorp/Enos-Docs)
 
 The test suite uses the [Playwright](https://playwright.dev/) framework.
 
+## Prerequisites:
+
+You will need [Hombrebrew](https://brew.sh/) install. For secure persisting your SSH keys and tokens we recommend using 1Password.
```

Comment:
```suggestion
You will need [Homebrew](https://brew.sh/) install. For secure persisting your SSH keys and tokens we recommend using 1Password.
```
though, homebrebrew sounds cool.

Reply:
:rofl: sorry to many brews. Thanks for the catch!